runtime/near-vm-runner/src/tests.rs

```diff
@@ -136,10 +133,26 @@ fn gas_and_error_match(
             assert_eq!(outcome.burnt_gas, gas, "burnt gas differs");
         }
         None => {
-            assert_eq!(outcome_and_error.0.used_gas, 0);
-            assert_eq!(outcome_and_error.0.burnt_gas, 0);
+            assert_eq!(outcome_and_error.0.used_gas, 0, "used gas non-zero");
+            assert_eq!(outcome_and_error.0.burnt_gas, 0, "burnt gas non-zero");
```

Comment:
Wait, why do we even make the `expected_gas` optional here? I think this can be just an `u64` now, after the first error refactor? Might be worth sending as a separate PR before this one, to separate simple change to a large number of tests. 

Also, I really don't like the shape of runtime tests, they are too chunrn-prone. Some day we need to make the tests here more data-driven...